docs/Removal.md

```diff
@@ -84,59 +51,37 @@ token id 0 URI points to vintage 2018 nori.com/api/removal/0 -&gt; { amount: 100
 token id 1 URI points to vintage 2019 nori.com/api/removal/1 -&gt; { amount: 10, supplier: 1, vintage: 2019, ... }
 token id 2 URI points to vintage 2020 nori.com/api/removal/2 -&gt; { amount: 50, supplier: 1, vintage: 2020, ... }_
 
-| Name | Type | Description |
-| ---- | ---- | ----------- |
-| to | address | The supplier address |
-| amounts | uint256[] | Each removal&#x27;s tonnes of CO2 formatted as wei |
-| vintages | uint256[] | The year for each removal |
-| data | bytes | Encodes the market contract address and a unique identifier for the parcel from whence these removals came. |
-
+| Name     | Type      | Description                                                                                                 |
+| -------- | --------- | ----------------------------------------------------------------------------------------------------------- |
+| to       | address   | The supplier address                                                                                        |
+| amounts  | uint256[] | Each removal&#x27;s tonnes of CO2 formatted as wei                                                          |
+| vintages | uint256[] | The year for each removal                                                                                   |
+| data     | bytes     | Encodes the market contract address and a unique identifier for the parcel from whence these removals came. |
 
 ### safeBatchTransferFrom
 
 ```solidity
 function safeBatchTransferFrom(address _from, address _to, uint256[] _ids, uint256[] _amounts, bytes _data) public
 ```
 
-
-
 _used to initiate a sale of removals by transferring the removals to the_
 
-
-
 ### supportsInterface
 
 ```solidity
 function supportsInterface(bytes4 interfaceId) public view returns (bool)
 ```
 
-
-
-
-
-
 ### vintage
 
 ```solidity
 function vintage(uint256 removalId) public view returns (struct Removal.Vintage)
 ```
 
-
-
 _returns the removal vintage data for a given removal token ID_
 
-
-
-### _beforeTokenTransfer
+### \_beforeTokenTransfer
```

Comment:
was this autogenerated by `hardhat docgen`?

Reply:
I never ran that but I think it was auto-generated because I didn't modify these files

Reply:
maybe just remove it from these changes from this branch? The `\_` doesnt seem like it's coming from docgen but could be wrong

Reply:
removed